packages/cli/src/databases/entities/AuthUser.ts

```diff
@@ -1,19 +1,19 @@
-import { Column, Entity, PrimaryColumn } from '@n8n/typeorm';
+import omit from 'lodash/omit';
+import { Column, Entity } from '@n8n/typeorm';
+import { User } from './User';
 
 @Entity({ name: 'user' })
-export class AuthUser {
-	@PrimaryColumn({ type: 'uuid', update: false })
-	id: string;
-
-	@Column({ type: String, update: false })
-	email: string;
-
-	@Column({ type: Boolean, default: false })
-	mfaEnabled: boolean;
-
+export class AuthUser extends User {
```

Comment:
Would be nice to have some reasoning about how is `AuthUser` different from `User` and how it should be used